specify/specify7
# Use global variable for directoryKey in global resources

Fixes #4683

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to App Resources -> Global Resources -> Any Preference (e.g. Global Preferences; https://fwri1924-edge.test.specifysystems.org/specify/resources/app-resource/1/)
- Click on Clone
- [ ] Verify the resource gets cloned. 
- [ ] proceed with several different type of resources
